waspy/transports/rabbitmqtransport.py

```diff
@@ -206,6 +207,10 @@ def get_client(self):
                                       queue_name=self.queue,
                                       routing_key=routing_key)
 
+    async def register_router(self, router, exchange='amq.topic'):
+        for topic in (parse_url_to_topic(*url) for url in router.urls):
+            await self.bind_to_exchange(exchange=exchange, routing_key=topic)
```

Comment:
not that it matters, because this is at startup, but these dont have to wait on eachother. So a `asyncio.gather` might be a little better

Reply:
I tried a gather but ran into issues with the library. I can troubleshoot that further if you feel it is worth it.

Reply:
Ya. I wonder what issues ran into. Woultod probably be good to get those issues marked, and possibly resolved.

Reply:
Looks like I was running into https://github.com/Polyconseil/aioamqp/issues/105

This seems to be an issue with our implementation. I see that we are also sharing a single channel for the client, we should open another channel for that.